Sources/SnowManLogic/GameLogic/Game.swift

```diff
@@ -0,0 +1,45 @@
+public class Game {
+	var word: String
+	var guessManager: GuessManager
+	var view: View
+	var guess: String
+
+	public init(word: String, guessManager: GuessManager, view: View) {
+		self.word = word
+		self.guessManager = guessManager
+		self.view = view
+		guess = ""
+	}
+
+	public func playGame() {
+		while (gameIsOver() == false) {
+			playerTurn()
+		}
+		view.displayResults(word, winner: isWinner())
```

Comment:
I was using the boolean to decide whether I display "you win" or "you lose". To fix this, I broke the `displayResults` function in the `View` to `displayWinningMessage` and `displayLosingMessage` and determined what to call in new `game.displayResults`